beers/models.py

```diff
@@ -0,0 +1,61 @@
+import string
+
+from django.db import models
+
+class BeerStyleCategory(models.Model):
+    CLASS_CHOICES = (
+        ('beer', 'Beer'),
+        ('cider', 'Cider'),
+        ('mead', 'Mead'),
+    )
+
+    name = models.CharField(max_length=100, unique=True, db_index=True)
+    bjcp_class = models.CharField(max_length=10, choices=CLASS_CHOICES, default='beer')
+    notes = models.TextField(default='')
+    category_id = models.CharField(max_length=2, unique=True)
```

Comment:
Stupid question: why is this a string? Shouldn't it be a number?

Reply:
I think because it was actually a string in the xml, because they differentiate between beer (1, 2, 3), Mead (M1, M2, M3) and Cider (C1, C2, C3).  I can strip the leading char and make it an integer since I already have the class there as well.

Reply:
Ah, that makes sense. If we're saving the beer/mead/cider separately, then making it an integer seems best to me.